src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -40,15 +40,15 @@ public FunctionsController(IWebFunctionsManager functionsManager, WebScriptHostM
 
         [HttpGet]
         [Route("admin/functions")]
-        [Authorize(Policy = PolicyNames.AdminAuthLevel)]
+        //[Authorize(Policy = PolicyNames.AdminAuthLevel)]
```

Comment:
Is this just temporary?

Reply:
yes, pending https://github.com/Azure/azure-functions-host/issues/2656. It's just so we can call these APIs from the UI until the auth is implemented.